lib/kramdown_rpf/version.rb

```diff
@@ -1,3 +1,3 @@
 module KramdownRPF
-  VERSION = '0.9.6'.freeze
+  VERSION = '0.9.7'.freeze
```

Comment:
Cue argument..

Would Semver say this is 0.10.0?

Reply:
a) yes
b) but anything goes in 0. versions

but yes